lib/elixir/test/elixir/string_test.exs

```diff
@@ -999,6 +999,7 @@ defmodule StringTest do
     assert String.jaro_distance("jon", "john") == 0.9166666666666666
     assert String.jaro_distance("jon", "jan") == 0.7777777777777777
     assert String.jaro_distance("семена", "стремя") == 0.6666666666666666
+    assert String.jaro_distance("Sunday", "Saturday") == 0.7194444444444444
```

Comment:
Adding the example mentioned [here](https://github.com/erlang/otp/pull/7863#issuecomment-1812776125)